qcengine/programs/pylibefp.py

```diff
@@ -0,0 +1,159 @@
+"""
+Calls the PylibEFP interface to LibEFP.
+"""
+import pprint
+from typing import Dict
+
+import qcelemental as qcel
+from qcelemental.models import Provenance, Result
+from qcelemental.util import safe_version, which_import
+
+from ..exceptions import InputError  #, RandomError, ResourceError, UnknownError
+from .model import ProgramHarness
+
+pp = pprint.PrettyPrinter(width=120, compact=True, indent=1)
+
+
+class PylibEFPHarness(ProgramHarness):
+
+    _defaults = {
+        "name": "PylibEFP",
+        "scratch": False,
```

Comment:
This is probably thread safe?

Reply:
all in memory and all functions acting on same struct instance, so yes.